62.CAD/main.cpp

```diff
@@ -1199,7 +1241,7 @@ class CADApp : public ApplicationBase
 	CPolyline bigPolyline;
 	CPolyline bigPolyline2;
 
-	bool fragmentShaderInterlockEnabled = false;
+	bool fragmentShaderInterlockEnabled = true;
```

Comment:
this will tell us if what we requested was enabled or not, should be false by default